app/src/main/res/values/strings.xml

```diff
@@ -1586,6 +1564,21 @@ Partially means that a wheelchair can enter and use the restroom, but no handrai
 
     <string name="select_overlay">Select Overlay</string>
 
+    <string name="overlay_cycleway">Cycleways</string>
+    <string name="bicycle_boulevard_is_a">It’s a %1$s</string>
+    <string name="bicycle_boulevard_is_not_a">It’s not a %1$s</string>
+    <string name="bicycle_boulevard">Bicycle boulevard</string>
+    <string name="separate_cycleway_no">No cycling</string>
+    <string name="separate_cycleway_allowed">Cyclists are allowed</string>
+    <string name="separate_cycleway_no_or_allowed">Not designated for cyclists (regardless of whether cycling is allowed or not) </string>
+    <string name="separate_cycleway_non_segregated">Designated combined foot- and cycleway</string>
+    <string name="separate_cycleway_segregated">Segregated cycle- and footway</string>
+    <string name="separate_cycleway_exclusive">Exclusive cycleway</string>
+    <string name="separate_cycleway_with_sidewalk">Exclusive cycleway with sidewalk (footway is segregated from cycleway by a kerb)</string>
```

Comment:
Maybe name it "cycleway has a sidewalk for foot traffic"?

So cases where it is clearly cycleway with sidewalk like https://community-cdn.openstreetmap.org/uploads/default/original/2X/0/0e3af26fdddf2ece75e97c6c0ca5c1e549e38273.jpeg from https://community.openstreetmap.org/t/deprecate-key-cyclestreet/6084/42 will be marked with this.

And infrastructure on edge between them, like https://wiki.openstreetmap.org/wiki/File:Witosa_bike_2.jpg will be tagged depending on local preferences (depending on is it considered as segregated footway+cycleway or cycleway with sidewalk)

And cases like https://commons.wikimedia.org/wiki/File:30111VoieVerte1248.JPG will be always tagged as segregated footway and cycleway

Maybe name it "wide cycleway that has a sidewalk for foot traffic"?

Reply:
What's wrong with the current string?

Reply:
Claims/implies that https://wiki.openstreetmap.org/wiki/File:Witosa_bike_2.jpg is mapped incorrectly if tagged

```
highway=path
segregated=yes
bicycle=designated
foot=designated
```

Reply:
Moving in circles here. It does not claim it is mapped incorrectly. "Segregated cycle- and footway" is correct, too.